src/main/java/org/isf/medicalinventory/gui/InventoryWardBrowser.java

```diff
@@ -82,8 +98,17 @@ public class InventoryWardBrowser extends ModalJFrame implements InventoryListen
             MessageBundle.getMessage("angal.common.user.col").toUpperCase()
     };
     private int[] pColumwidth = { 150, 150, 100, 100, 150 };
-    private JComboBox<Object> stateComboBox;
-    private JLabel stateLabel;
+    private JComboBox<String> statusComboBox;
+    private JLabel statusLabel;
+    JButton next;
+    JButton previous;
+    JComboBox pagesCombo = new JComboBox();
```

Comment:
Why doesn't isn't there a parameterized type for this generic object (see line 101 for example)